### PR TITLE
[TW-1652] Minor wording change in first-steps/overview

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**

### CODE REVIEW

File: src/pages/docs/getting-started/first-steps/overview.md [hunk 1]
```diff
@@ -37,6 +37,6 @@ contextual_links:
 
 Follow this quick guide to get started using Postman. You'll learn how to install the Postman app, send an API request, write your first test, and save requests in a collection. You'll also learn about working with your team in Postman and discovering APIs on the Public API Network.
 
-When you're ready to begin, select **Download** in the navigation menu at left. Postman recommends going through each of the topics in this section in order. Make sure to sign up for a free Postman account when directed so you can save your work.
+When you're ready to begin, select **Download** in the navigation menu at the left. Postman recommends going through each of the topics in this section in order. Make sure to sign up for a free Postman account when directed so you can save your work.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/first-steps/overview.md:40]:
```suggestion
When you're ready to begin, select **Download** in the navigation menu on the left. Postman recommends going through each of the topics in this section in order. Make sure to sign up for a free Postman account when directed so you can save your work.
```

**Review by mark-anthony-postman — APPROVED** [review]
Changed it to "on" per our discussion. Thanks! 😀

File: src/pages/docs/getting-started/first-steps/overview.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/first-steps/overview.md:40]: Thanks!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**